server/index.js

```diff
@@ -42,9 +45,9 @@ if (config.server.env === 'DEV') {
 
 app.use(errorHandleMiddleware)
 
-app.get('/', (req, res) => {
-  res.send('Hello, world!')
-})
+app.get('*', (req, res) => {
+  res.sendFile(path.join(__dirname, '../client/build', 'index.html'));
+});
```

Comment:
does it make a difference if we change from '/' to '*'?

Reply:
'/' is the root URL, '*' is all other path that was not defined in our backend